.storybook/preview.js

```diff
@@ -6,7 +6,6 @@ import { withA11y } from '@storybook/addon-a11y';
 import MockAppDecorator from './addons/MockAppDecorator/MockAppDecorator'
 import qzTheme from './themes/qz';
 
-addDecorator( withSmartKnobs() );
```

Comment:
I spent a while trying to figure out how to get this to work with docs, but to no avail. It does work if you declare your own knobs, but that's kind of a drag